src/cmdlib.sh

```diff
@@ -288,6 +287,47 @@ commit_overlay() {
         --skip-list <(echo /statoverride)
 }
 
+create_content_manifest(){
+    mkdir -p "${workdir}"/tmp/buildinfo
+    rpm-ostree compose tree --repo="${tmprepo}" --print-only "${manifest}" > "${manifest_tmp_json}"
```

Comment:
Let's set `manifest_tmp_json` ourselves here as a `local` variable (and the `manifest_tmp_json` in `prepare_build` should probably also be `local`).

Reply:
Alternatively, we can formalize this as part of `prepare_build` and not delete the treefile like you had before but name the variable something like `flattened_manifest` and explicitly `export` it the same way we export `manifest`. Then here we can reuse it instead of rerunning `rpm-ostree compose tree`. Anyway, not a blocker!